qemu/tests/cfg/virtio_port_hotplug.cfg

```diff
@@ -37,6 +37,12 @@
             filesize = 100
             unplug_chardev_vc1 = no
             unplug_chardev_vc2 = no
+        - unplug_port_chardev_pci:
+            type = virtio_serial_unplug_port_chardev_pci
+            sleep_time = 0
```

Comment:
For linux guest,  `0` is enough ?  I see some other chardev hot plug may wait for `0.5s` then plug the virtio port to this chardev.

Reply:
ok, let's set the default value to 0.5s